comp/core/tagger/impl-remote/remote.go

```diff
@@ -108,11 +84,8 @@ type remoteTagger struct {
 	telemetryTicker *time.Ticker
 	telemetryStore  *telemetry.Store
 
-	datadogConfig taggercommon.DatadogConfig
```

Comment:
Could we move the `DatadogConfig` defintion from the common package to the local tagger implementation? 

Since is no longer use by multiple implementation we do not need it to be public 😄